src/main/java/net/mohron/skyclaims/database/Database.java

```diff
@@ -35,24 +35,32 @@
 
 public abstract class Database implements IDatabase {
 	abstract Connection getConnection() throws SQLException;
+	private String tablename = "island";
 
 	/***
 	 * Creates a table using SQL syntax for Sqlite or Mysql
 	 */
 	void createTable() {
+		String type = SkyClaims.getInstance().getConfig().getStorageConfig().getType();
+
+		if(type.equals("MySQL")){
+			this.tablename =  SkyClaims.getInstance().getConfig().getStorageConfig().getMysqlConfig().getTablePrefix() + "island";
+		}
+
 		try (Statement statement = getConnection().createStatement()) {
 			statement.setQueryTimeout(30);
 
 			// Create the database schema
-			String table = "CREATE TABLE IF NOT EXISTS islands (" +
-					"island			STRING PRIMARY KEY," +
-					"owner			STRING," +
-					"claim			STRING," +
-					"spawnX			INT," +
-					"spawnY			INT," +
-					"spawnZ			INT," +
-					"locked			BOOLEAN" +
-					")";
+			String table =
+				"CREATE TABLE IF NOT EXISTS `" + this.tablename + "` ( " +
+				"`island` TEXT NOT NULL, "+
+				"`owner` TEXT NOT NULL, " +
+				"`claim` TEXT NOT NULL, " +
+				"`spawnX` INT NOT NULL, " +
+				"`spawnY` INT NOT NULL, " +
+				"`spawnZ` INT NOT NULL, " +
+				"`locked` BOOLEAN NOT NULL, " +
+				"PRIMARY KEY " + (type.equals("SQLite") ? "(`island`)" : "(`island`(60))") + ");";
```

Comment:
I couldn't find documentation of that for some reason lol. Why 60 then? UUIDs are a fixed length, 36 with dashes, https://en.m.wikipedia.org/wiki/Universally_unique_identifier